extensions/projection/README.md

```diff
@@ -94,3 +121,38 @@ This object represents the centroid of the Item Geometry.
 | ------------------- | ------ | ------------------------------------------------------------ |
 | lat                 | number | The latitude of the centroid.  |
 | lon                 | number | The longitude of the centroid. |
+
+## Best Practices
+
+There are clearly a number of projection extension fields, with potentially overlapping functionality. This section attempts to 
```

Comment:
==> "There are a number of projection extension fields with potentially overlapping functionality."